blog: v6.2 announcement (#5359)
simplex-chat/simplex-chat: blog/20241210-simplex-network-v6-2-servers-by-flux-business-chats.md

SimpleX Chat and [Flux](https://runonflux.com) (Influx Technology Limited) made an agreement to include messaging and file servers operated by Flux into the app.

SimpleX network is decentralized by design, but in the previous versions users had to find other servers online or host servers themselves to use servers other than operated by us.

Now all users can choose between servers of two companies, use both of them, and continue using any other servers.

To use Flux servers you have to enable them when the app offers them or at any point later via Network & servers settings in the app.

When both SimpleX Chat and Flux servers are enabled, the app will choose servers of different operators in each connection to receive messages and for [private message routing](./20240604-simplex-chat-v5.8-private-message-routing-chat-themes.md), increasing metadata privacy for all users.

Read more about why SimpleX network has to have multiple operators in [our previous post](./20241125-servers-operated-by-flux-true-privacy-and-decentralization-for-all-users.md).

> You can also readabout our plan [how network operators will make money](https://github.com/simplex-chat/simplex-chat/blob/stable/docs/rfcs/2024-04-26-commercial-model.md), while continuing to protect users privacy, based on network design rather than on trust to operators, and without any cryptocurrency emission.

Reviewer: ```suggestion
You can also read about our plan [how network operators will make money](https://github.com/simplex-chat/simplex-chat/blob/stable/docs/rfcs/2024-04-26-commercial-model.md), while continuing to protect users privacy, based on network design rather than on trust to operators, and without any cryptocurrency emission.
```

### Business chats

<img src="./images/20241210-business.png" width=288 class="float-to-right">

We use SimpleX Chat to provide support to SimpleX Chat users, and we also see some other organizations offering SimpleX Chat as a support channel.

The problem of providing support via general purpose messaging platforms is that the customers prefer knowing when they talk to a bot, and when to a human, and also who they are talking with.

It is not possible to achieve with the usual contact address in SimpleX Chat, as it creates a new one-to-one conversation with every new contact.

Business chats solve this problem: all you have to do is to enable the toggle under the contact address in your chat profile.

> Once you do it, the application will be creating a new business conversation with each customer where multiple people can participate. Business conversation is a hybrid of one-to-one and group conversation. In the list of chats you will see customer name and avatar, and the customer will see business name and avatar. But inside it works as a group, allowing cuatomer to see who sent the message, and allowing you to add other participants from the business side, for delegation and escalation of customer questions.

Reviewer: ```suggestion
Once you do it, the application will be creating a new business conversation with each customer where multiple people can participate. Business conversation is a hybrid of one-to-one and group conversation. In the list of chats you will see customer name and avatar, and the customer will see business name and avatar. But inside it works as a group, allowing customer to see who sent the message, and allowing you to add other participants from the business side, for delegation and escalation of customer questions.
```

### Better user experience

<img src="./images/20241210-reactions.png" width=288 class="float-to-right">

**Chat navigation**

This has been a long-standing complaint from the users: *why does the app opens conversations on the last message, and not on the first unread message*.

This is how it works now in Android and desktop apps, and will soon be done in the iOS app too. 

Also, the app can scroll to the replied message when you tap it anywhere in the conversation, even if it was sent a very long time ago

**See who reacted!**

This is a small but important change - you can now see who reacted to your messages!

### Improving notifications in iOS app

iOS notifications in a decentralized network is a complex problems. We [support iOS notifications](./20220404-simplex-chat-instant-notifications.md#our-ios-approach-has-one-trade-off) from early versions of the app, focussing on preserving privacy as much as possible. But the reliability of notifications was not good enough.

We solved several problems of notification delivery in this release:

> - messaging servers no longer nowlose notificaitons while notification servers are restarted.

Reviewer: ```suggestion
- messaging servers no longer now lose notificaitons while notification servers are restarted.
```
